### PR TITLE
10192 add module timesheet_activity_report

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by duanyp1991** [comment]
LGTM

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: timesheet_activity_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20timesheet_activity_report%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Sébastien Maillard <sebastien.maillard@elico-corp.com>
+
```
Comment (elicoidal) [inline timesheet_activity_report/README.rst:37]: Eric Caudal eric.caudal@elico-corp.com

**Comment by elicoidal** [comment]
add contributor. Other than that LGTM

**Comment by Rona111** [comment]
cc @elicoidal  @duanyp1991  @seb-elico @siyuancn

**Comment by siyuancn** [comment]
LGTM

**Pull request merged**